events/break-free/index.html

```diff
@@ -72,6 +72,8 @@
             {{ description }}
           </div>
         </script>
+
+  <script type="text/javascript" src="js/action-kit-service.js"></script>

```

Comment:
In the next PR after this one gets merged, you'll be able to see all the dependency management refactoring we did. We can definitely try bundling in that PR as well, or in a future one.